tests/test_content_client.py

```diff
@@ -104,8 +104,17 @@ def test_available_errata(katello_client, entities, api, os_release_major):
     katello_client.run('yum -y install walrus-0.71')
 
     host = foreman_host(katello_client, api)
-    errata = api.resource('errata').call('index', {'host_id': host['id']})
-    errata_ids = [erratum['errata_id'] for erratum in errata['results']]
+
+    retries = 0
+    while retries < 5:
+        retries += 1
```

Comment:
Isn't this equal to:
```suggestion
for attempt in range(5):
```